airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -293,6 +290,7 @@ abstract class BasicFunctionalityIntegrationTest(
     val isStreamSchemaRetroactiveForUnknownTypeToString: Boolean = true,
     val dedupBehavior: DedupBehavior?,
     val stringifySchemalessObjects: Boolean,
+    val stringifyUnionObjects: Boolean = false,
```

Comment:
do we still need this? I thought we avoided it by changing how JSON values get written

Reply:
ah, got it. I had a question about why the behavior is different in the other thread